src/engine/strat_engine/filesystem.rs

```diff
@@ -150,19 +150,38 @@ impl StratFilesystem {
     }
 
     /// Get the mount_point for this filesystem
-    /// TODO Replace this code with something less brittle
     pub fn get_mount_point(&self) -> EngineResult<Option<PathBuf>> {
-        let output = Command::new("df")
-            .arg("--output=target")
-            .arg(&self.devnode())
-            .output()?;
-        if output.status.success() {
-            let output_str = String::from_utf8_lossy(&output.stdout);
-            return Ok(output_str.lines().last().map(PathBuf::from));
```

Comment:
I don't know if that matters but here you were getting the last match…

Reply:
No, it shouldn't matter.  Thanks for the thorough review!